SI/Sem.02/Pract.02/Task3.cpp

```diff
@@ -0,0 +1,14 @@
+// ConsoleApplicationPrakt2Task3.cpp : This file contains the 'main' function. Program execution begins and ends there.
+//
+
+#include <iostream>
+using namespace std;
+int main()
+{
+	int number1, number2;
+	cout << "Enter two numbers"<<endl;
+	cin >> number1>> number2;
+	cout << ((number1 != number2 && number1 > number2) ? "The first one is larger" : "The second one is smaller");
```

Comment:
If number1 > number2, they are never equal, so you do not need to check number1 != number2. "The second one is smaller" will be printed if the condition is false, which means number1 <= number2, but in this case the second is bigger. Also, you need to make sure the numbers are different in both cases.